detection-rules/generic_service_abuse_reply_to.yml

```
@@ -0,0 +1,27 @@
+name: "Generic Service Abuse From Newly Registered Domain"
+description: "Detects messages from services that write the true sender to the reply-to field, where the sender has no prior legitimate message history and is newly registered. Indicative of service abuse."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(headers.reply_to, network.whois(.email.domain).days_old < 30)
+  and sender.email.domain.domain in $replyto_service_domains
+  // 
+  // This rule makes use of a beta feature and is subject to change without notice
+  // using the beta feature in custom rules is not suggested until it has been formally released
+  // 
+  and not beta.profile.by_reply_to().solicited
+  and not beta.profile.by_reply_to().any_messages_benign
+
```

thoughts on using reply_to_email()? 
```suggestion
  // 
  and not beta.profile.by_reply_to_email().solicited
  and not beta.profile.by_reply_to_email().any_messages_benign

```

---

Probably not needed here because the logic of the rule is on the domain of the reply_to address.